servers/maxstream.py

```diff
@@ -9,106 +7,5 @@
v5g1FX7Xz4hwGDyIuFY7Ms0X9lwpJGm7/ojClTqPnSpkMP4kad5hJV7FZpmTJBFwixx8EdwQ15rgTbS8ZwrogG78VLkPMVU+vrII2Jq/8TArELc1QVUmdgUW8t6vHCr4rVLijbfkOv2GfhxWSiJLSRV1rq6xBHeQodKqTFgkaKQ9IsCpX/nFCD9CPQPe3+0IK+q4B3xAI97TClS3bNYGg0IxpKwCdxI/HEkAG8Byd2jCOCTPN7Yogg1cKNzyzz2HvdBpSf01dnHb9+HXiNaGd+ZBy6iDKEYWs8ssOZzBvbRRNQW1RthxkZLTQ5T4YB4XsXkG8hPANqaKXU6ePHW8lZwFN3wpRDiyIzE+HQbwK/FJxdzi7B+aUgouxRAzBJyk++F1RVmsO6I+aNy4/bxKBqCxkhVtuyNfMlWLRUNjVBr0UMGJ+KJjkKFIE58tdn+URW+cnLylwCyhAXurrAnN/vgBCp5HvyqQYWTq4kfjQfh60CqXjSAaVsXsr22JQcS0u6p3r8KSQXqKa4vfSzrE/0l+aS41mvWaf8d6Kw1HlYR44rLz09+ZlmpOdRnT+RrpE30R29Sl6PsrYQ6FZDH4NIMeYSkGYG2VX+6s62IvetTe9jZvwR2UIA7eLUoSEN5KYIU3b0WdWXjqpCmHPZ/9SneyevXIojt7zr1yZ032SngL8x00IdJz+IGnLjzC2AxdOEhpyOeZbGyPI4TapsfijLB+knkXi8RhiTcIWQRVIC723wTSRGaAlTiZGiryY45qBVsmhK1Pa+e+rw63sAGDZpPYl1HpnJYwWx6N2VFQAOF0Rc4ZYJwifqdARF8XFC+364820763W8SywKBCJjj3OOnHOh3WqsJn3N7ukxkVh6nkcW+qHXNMiqBKGyeFR0SrCqD/IXFC2JPJMqMQUNYcWIoymtiNT/BZdt+5+4JgKtAWNj+INtpgYVCFoblAdnhN7NKJZrJ8HROC43StPPy3Rc8R1xSV1gEw2C8tCHPXtehCI4RyksyhUnGMoK4vxuhlgXgQjR0BVPFm2Nqmk1wb37mSFmT/RfpFJgLkWP45FvbY6Ntg3NVaeTkjZjPKMgemVH/2ApmM87lAWB8STWdSJWZvFOzAORNjOPFOHJ0ZGLy0EbpGIoa755RCsgl+RctLLWHYIx48hqe13/C4OUjvhXKesMGApU4sIyE/0lvI6VkNyEDagIcwGcCnMsuPDm7sn6MVBpJGJD/5wwDzSjrmE5BHx4aqvJVDfbcsjao6ikmiJyYiiHMs4oG+goNdAnWeaznlV7mwjcfEksJk7HpymwDVuOJZ0JJ2GKAiUSE3TSIpGGjamgDFWN/RqckMuFmCYfDwn8xhEuIfwEFVXSu+xCIjYgaBnhf7tg0HXQMnx/jNodL0dLVfCl+toUNgHu/wDKS8ePsO3CihcZBIdPo1HU2U4yvb2m1XNylRMQxU2Ut1r5KJTdHD+IKlxHFASAdeLp1cs8Jlx/+Dfaq+46ISu7T+3ZSD+w8rGbpSpsPqkdYKpCeEiximrN8Mxu2uc8YVsFWg0nO92oU/roTWwbbRzRwxvwa5ytlAuY/1UAk6/9sBI1JkX7h3cW/ewzuG7Rt82H45ogidzY/p6FOWVSkgMGCbDt1+GWK4qfGqdsyw760GmAUKYkFN4ePL1NxTnHmTrRcumDyzz4NTD6brI7K/jYt55o9fIdDhoRUxktVgiD91mbJnRL0zo59VyEuubqLikSEyH8tf8BeMxXaFizZI2XED5gq6Vce7+D2O9HRqPYWDjkCClkvHTJpRpzeT8MpJcUbk1LsFnOnM0loXKPfDnrp5CZnAhJTAUHGpVLUuALl8+gGMOMk5Cm8iI1Lsug6oJdwIUG96UtimhV08L5DP48Z0oZ7iXVyyz9k785KL2ZT/Alu0K/WJ86TXfT2EMrwH5qtCS2HBtpGysiruTMaSp7twxJc/T0bmBRZM8Fgaf6/lU9ENIRDJ1eNRGPcD1P1VSf7p/qmZ0HlO4PJSmWVh4Pi1fw+RIxVAuMLTHpLuLcdH4yjx1Yh0OiMaHmsybd58fByzPtZaBw7muUKLSYniZzcTvfjIkFHiAF9bgTS+HRXWyDCJgzVCL6BkCeFo222742KCxkHdjSi4oXkVzSIOc7FhZrWYYX5mI/UIqSQhA3DmJAxbaI5I8GoMlu7sZAgGbl3w7DdLv6koSfx97W+ZLFWUCDISnkhrP6DiLWoeniS8bvU0byBuS8VbHcZnREVujxkvPisZOdsRzoKnDtXj46nqpbGwFU9rOhlOD71F7onXBhHYLTkBqj1NCFDPQ20Nh9o4Yg+nN8EXtdqboVlvOK1AS6bW8KaB68GlnhbJ96goCdvRjnSxzCqBWRooV8dhREYyLI57SIzuG4onFI/VqS9Mjy8J33RSC5k2foCwKc47bTsEgv8Xm1WvB74UiBM9nu5uD2M7Oj5+Q02FUy2CfkrEGGEUDMJ+gpkfWtTCg4vD1tY+18PhKwlQN9ldf9gt/So30YN9g6ycWNEeCLrZrjClY+9mp8jX6Hz0zq2zwJJuzWma6KhajbcpXA3WceQJM9l1859lmuSTcHQOiYLNWXEWovxk+eoJGZz+kJtasP1uK8n9tb/npBzroUXdeS0zxVdQ3CWgsxTACy33ogZcX4plfXC2RVWOIOEuGUNan1kSnl79O+HWN+TFZ2uKwJ5wHOfOPB5oD5iCNu6E3RagkJUd+8qKaTqW/hW3rBokst94k+CLEAMdypSsmUefeMUzCaxtE//xCaLRk59y+vv6rRHONo1JroO1Zb2jxLG87sWhD+Rh9oQSk1Gj9/xBKDaCMPPgh5xSLtP9aEAtAKu7+qlpfB3R3DQnJvrVBur1jyRlhEIX7jVVhgJCrQqg32Z9v61qXJd2GPOzfjJYRA28LQdffhwxsJtpiMtsaJTskBFwhTz7mj93z7fS/+//zz/fmPZ5TrsiqjO6os98vflVmIyG6XxMx1wImGK0Zn9DRSgUxyW7lNwJe'))
```

Comment:
Riflettendoci su, forse non è il caso di tenerlo offuscato, avevo fatto un tentativo perchè c'era il webmaster che palesemente guardava il codice e provava a modificare l'html del sito in modo da fargli leggere il link di un porno gay...
Anche se fosse offuscato pesantemente proverebbe comunque a spostare/rinominare codice finchè vede che KoD non fallisce. Dobbiamo solo sperare che non ricominci

Reply:
Okk, speriamo..